.travis.yml

```diff
@@ -12,7 +12,7 @@ language: go
 
 env:
   global:
-    - GO_METALINTER_VERSION="v3.0.0"
+    - GOLANGCI_VERSION="v1.15.0"
```

Comment:
you can use the latest version ie v1.16.0

Reply:
Done